modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/FragmentPlan.java

```diff
@@ -53,7 +53,7 @@ public Type type() {
     /** {@inheritDoc} */
     @Override
     public ResultSetMetadata metadata() {
-        return List::of;
+        return ResultSetMetadataImpl.NO_METADATA;
```

Comment:
I wonder if anyone asking fragment about its meta. Probably it would be better to throw IllegaleState here, WDYT?